docs/en/source/advanced/quantization.rst

```diff
@@ -0,0 +1,34 @@
+*************************************
+Model Quantization
+*************************************
+
+Warboy is a FuriosaAI NPU that only supports 8-bit integer models.
+To support models based on real data types, the Furiosa SDK provides a tool that quantizes*
```

Comment:
I think we don't have to translate exactly Korean to English in line by line. We can reorganize this paragraphs to be more natural in English. Let me think about too.